main.go

```diff
@@ -93,6 +93,11 @@ func main() {
 	}
 
 	// Migrate the schema
+	err = migration.Migrate(db)
+	if err != nil {
+		panic(err.Error())
```

Comment:
It sounds like 'serializable transactions' will result in a controlled version of what I describe over. Both will do the work, but the first one to finish will pass while the second will fail because they try to change the same thing. Since a migration could be a fairly heavy task on the database, wouldn't it be preferred if it was only attempted once?

Since this is a 1 time operation during startup/deploy I don't think 'lock performance' matters here.